### ISSUE TITLE
Legend labels inaccessible via keyboard navigation

### ISSUE THREAD

**Issue by shainanigans** [swimlane/ngx-charts#1329]
**Describe the bug**
Users who use only keyboard to navigate cannot access the labels using the keyboard. This fails Level A accessibility compliance [2.1.1](https://www.w3.org/WAI/WCAG21/quickref/#qr-keyboard-operation-keyboard-operable).

**To Reproduce**
1. Go to [this demo](https://stackblitz.com/edit/swimlane-normalized-horizontal-bar-chart?embed=1&file=app/app.component.ts)
2. Click inside the browser panel
3. Tab through the elements with the tab key
4. Legend labels cannot receive focus

Looking at the markup of this component, it uses a `<span>` when it should be a `<button>` (at least if `legendLabelClick` events are being used).

**Expected behavior**
User should be able to access labels with keyboard when there are events attached to them.
